openapi/src/parts/alpha1.yaml

```diff
@@ -4,6 +4,52 @@
 ## may change without notice.
 
 paths:
+  /api/workspaces/alpha1/{workspaceId}/resources/controlled/gcp/buckets/{resourceId}/load:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/ResourceId'
+    post:
+      operationId: loadSignedUrlList
```

Comment:
i suggest we avoid "signed" in the naming of these since these are really just urls.  they don't have to be signed.  they just have to be accessible.